lua/src/macro_cache.cc

```diff
@@ -188,6 +257,34 @@ void macro_cache::_process_host(neb::host const& h) {
   _hosts[h.host_id] = h;
 }
 
+/**
+ *  Process a host group event.
+ *
+ *  @param hg  The event.
+ */
+void macro_cache::_process_host_group(neb::host_group const& hg) {
+  logging::debug(logging::medium)
+    << "macro_cache: process host group --- "
```

Comment:
Could you normalize the log message ? The descriptor should be _lua:_ and properties are usually provided within parenthesis (check the SQL module for more examples).